Makefile

```diff
@@ -0,0 +1,14 @@
+all: fmt build
```

Comment:
I don't think `all` should depend on `fmt`: a user building the project just wants to build the project, not reformat the code.

Reply:
Made it independent for now. Normally it is very fast and since normally CI enforces it sooner or later anyway, many of us find it convenient to have it tied.